CMakeLists.txt

```diff
@@ -1,11 +1,17 @@
 cmake_minimum_required (VERSION 2.6)
-project (LIBUAST C)
+cmake_policy(SET CMP0048 NEW)
+project (LIBUAST
+  VERSION 1.4.0
```

Comment:
* This should be 0.1.0 at this point I think. There's been on public release yet.
* It makes sense to make a placeholeder though, since it should be updated by travis, something like `%%VERSION%%` or similar